### PR TITLE
Language

### PR DESCRIPTION
**Pull request by MrLarso2002**

### CODE REVIEW

File: src/blocks/invention/fsh-api-animal.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Blockly from "blockly/core";
 const blockName = "inv_fsh_api_animal";
 
 const blockData = {
-  "message0": "get %1 image from fsh api",
+  "message0": "gets %1 of the possible images from the fsh api",
```
Comment (Malix-Labs) [inline src/blocks/invention/fsh-api-animal.js]: Checking sementic from https://github.com/scratch-for-discord/Web-Application_Frontend/discussions/203

Comment (JeremyGamer13) [reply src/blocks/invention/fsh-api-animal.js]: image
the block makes less sense now?
it grabs a random image of the specified animal

Comment (MrLarso2002) [reply src/blocks/invention/fsh-api-animal.js]: i just noticed wait

File: src/blocks/invention/fsh-api-animal.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import Blockly from "blockly/core";
 const blockName = "inv_fsh_api_animal";
 
 const blockData = {
-  "message0": "get %1 image from fsh api",
+  "message0": "get %1 image from the fsh api",
```
Comment (Malix-Labs) [inline src/blocks/invention/fsh-api-animal.js]:
```suggestion
  "message0": "get a random image link of %1 from the fsh api",
```

File: src/blocks/invention/fsh-api-animal.js (hunk 1 above)
Comment (MrLarso2002) [reply src/blocks/invention/fsh-api-animal.js]: fixed in https://github.com/scratch-for-discord/Web-Application_Frontend/pull/202/commits/70823d6ad3d1c66157b478c45d9ae4d21c509aca

Comment (Malix-Labs) [reply src/blocks/invention/fsh-api-animal.js]: https://github.com/scratch-for-discord/Web-Application_Frontend/pull/202#discussion_r1133144296 makes a bit more sense to me

**Pull request merged**